app/components/security/analysis-list/table/name/index.hbs

```diff
@@ -0,0 +1,10 @@
+{{#if @analysis.id}}
```

Comment:
is there a no `id` scenario ?

Reply:
Yeah. When adding an analysis to the list, the analysis is first created via the store. This would lead to the addition of the analysis in the analysis list table. Since no ID would exist for the analysis until submission, it breaks the UI on the vulnerability name column which requires that ID for creating the link to the analysis details page.